server/chat-plugins/chatlog.ts

```diff
@@ -660,6 +703,8 @@ export const commands: ChatCommands = {
 		"/searchlogs [search], [room], [cap], [date] - searches logs in the current room for [search].",
 		"A comma can be used to search for multiple words in a single line - in the format arg1, arg2, etc.",
 		"If a [cap] is given, limits it to only that many lines. Defaults to 500.",
+		"The delimiter | can be used to space searching for multiple terms.",
```

Comment:
Weird to use `|` when you're searching for all the terms, not any of the terms.

Reply:
Ideally, it should support `,`.

Reply:
I decided on | because `,` is more commonly used for different params, so i decided it'd make more sense to have `/sl [queries], [room]...etc` as opposed to some complicated way to sort through the `,`s to find the data.

Reply:
I'll work on improving the system for this, though.

Reply:
Yeah, I know, we're already using `,` to separate parameters.

I don't really like `|` to separate parameters but we already have other commands that use it that way. :/ I think I'd much rather do it the other way around (using `|` to give room IDs and search limits and date ranges), because searching for multiple things is probably generally more useful.

Reply:
(fwiw, I suggested using `+` to separate the first argument, since it more easily communicates a combined search)

Reply:
I'll go with that.

Reply:
I believe dexsearch uses `,`? It might be better to be consistent with that.

I just don't think searching for rooms (rather than just `/join`ing the room) and having custom limits/dates is worth taking up the usual `,` symbol.

Reply:
I've reflected, and I think I like using `limit:` etc as prefixes.

Something like:

`/sl hello, zarel, limit: 30, date: 2020-03`

That would also help you not need to memorize the argument order.

Reply:
Will do.